### PR TITLE
improve operators placement docs

### PR DESCRIPTION
**Pull request by imanjra**
## What changes are proposed in this pull request?

(Please fill in changes proposed in this fix)

## How is this patch tested? If it is not, please explain why.

(Details)

### Is this a user-facing change that should be mentioned in the release notes?

-   [ ] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

File: docs/source/plugins/index.rst [hunk 1]
```diff
@@ -146,10 +146,58 @@ Regardless of operator type, all operators can be executed from code. For exampl
...
+  - DISPLAY_OPTIONS
+
+
+To add a placement for an operator to one of these places in the app, your
+operator class much implement method `resolve_placement` as demonstrated below:
+
+**Python Operator**
+.. code-block:: python
+
+    def resolve_placement(self, ctx):
```
Comment (ritch) [inline docs/source/plugins/index.rst:170]: @brimoor is there a way to show multiple code examples with a tab to select the language?

**Review by ritch — APPROVED** [review]

**Comment by brimoor** [comment]
@imanjra you can use `.. tabs::`: https://github.com/voxel51/fiftyone/blob/660545dcbc4ad3883cc9bb59f53c212b5e85e628/docs/source/user_guide/brain.rst?plain=1#L536

File: docs/source/plugins/index.rst (hunk 1 above)
Comment (brimoor) [reply docs/source/plugins/index.rst:170]: yeah @imanjra you can use `.. tabs::`: https://github.com/voxel51/fiftyone/blob/660545dcbc4ad3883cc9bb59f53c212b5e85e628/docs/source/user_guide/brain.rst?plain=1#L536

Comment (imanjra) [reply docs/source/plugins/index.rst:170]: Nice! Let me give it a try

**Pull request merged**